controllers/articles.controller.js

```diff
@@ -0,0 +1,17 @@
+const { selectArticleById } = require("../models/articles.model");
+
+function getArticleById(req, res, next) {
+  const { article_id } = req.params;
+  selectArticleById(article_id)
+    .then(({ rows }) => {
+      if (rows.length === 0) {
+        return Promise.reject({ status: 404, msg: "Article does not exist" });
+      }
```

Comment:
This logic should be in your model. And as you are dealing with just one article object, you can get the model to return the first item from the array. There is no need to respond with the array